apps/client/src/features/bookmark/bookmar-modal-import/ui/bookmark-modal-import.tsx

```diff
@@ -0,0 +1,115 @@
+import type { ImportBookmarkForm } from "@/entities/bookmark";
+import { Button } from "@/shared/ui/button";
+import { trpc } from "@/shared/trpc";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { FormProvider, useForm } from "react-hook-form";
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@/shared/ui/dialog";
+import { ImportBookmarkFormUi, importBookmarkSchema } from "@/entities/bookmark";
```

Comment:
Rename `ImportBookmarkFormUi` to just `ImportBookmarkForm` please.

Reply:
Actually, I have used the `ImportBookmarkForm` name for the Zod schema. To keep it in sync with the `Form` we use to import individual bookmarks.

Updated the name this way:
ImportBookmarkForm -> ImportForm (Zod Schema)
ImportBookmarkFormUi -> ImportBookmarkForm